chapters/6-analysis.tex

```diff
@@ -147,13 +147,19 @@ \subsection{Fitting construction into the data workflow}
 in a unified script, separate from the analysis code, helps
 avoid this and ensures consistency across different outputs.
 
-\subsection{Integrating different data sources}
+\subsection{Combining different data sources}
 
 % When merging is necessary and how to start thinking about it
 To create the analysis dataset,
-it is typically necessary to combine or merge information
-from different data sources.\sidenote{
-	\url{https://dimewiki.worldbank.org/Data_Integration}}
+it is typically necessary to combine information
+from different data sources.
+As discussed in Chapter 3, 
+this process should be documented and 
+communicated about using \textbf{data flowcharts},\sidenote{
```

Comment:
```suggestion
this process should be documented using \textbf{data flowcharts},\sidenote{
```

Reply:
done in 74e1ac0a2fe9f1d90ee32d95431aefe588060c63